samples/local-service-registry/README.md

```diff
@@ -33,13 +33,17 @@ Please find the details as follows.
 ```java
     public class xxxClient {
       public static void main(String[] args) {
+      //"yourpath" must be used absolute path
 　　    System.setProperty("local.registry.file", "/yourpath/registry.yaml");
-        //your code
+
+      //your code
+      ......
+
 　　    System.clearProperty("local.registry.file");
```

Comment:
clear is not necessary?

Reply:
process will exit, clear or not, have the same effect？

Reply:
ok, clearProperty can be deleted